src/extension/background-script/actions/alby/isEnabled.ts

```diff
@@ -0,0 +1,30 @@
+import { getHostFromSender } from "~/common/utils/helpers";
+import db from "~/extension/background-script/db";
+import type { MessageAllowanceEnable, Sender } from "~/types";
+
+import state from "../../state";
+
+const isEnabled = async (message: MessageAllowanceEnable, sender: Sender) => {
+  const host = getHostFromSender(sender);
```

Comment:
@bumi do you think it would be ok to have a function that just takes the provider name and then all actions can use it? 

The only difference between these files is "alby" vs "liquid" etc.